### PR TITLE
DO NOT MERGE: Main screen transparency switch through theme selection

### PR DESCRIPTION
**Pull request by rsudev**
In order to circumvent issues in the latest version of Android support in Sailfish OS (see #8752), switch the transparency on the main screen through theme selection.
In addition does this version try to detect SFOS and switches transparency off by default in this case.
This PR also disables leak canary, to make working with it as a user a bit smoother.
So do not merge this PR in any case, I'll prepare one without the latter change in case of successful testing.

### CODE REVIEW

File: main/src/cgeo/geocaching/settings/Settings.java [hunk 1]
```diff
@@ -85,6 +85,16 @@
     @NonNull private static final String TWITTER_KEY_CONSUMER_PUBLIC = CryptUtils.rot13("ESnsCvAv3kEupF1GCR3jGj");
     @NonNull private static final String TWITTER_KEY_CONSUMER_SECRET = CryptUtils.rot13("7vQWceACV9umEjJucmlpFe9FCMZSeqIqfkQ2BnhV9x");
 
+    private static final boolean TRANSPARENT_BACKGROUND_DEFAULT;
+
+    static {
+        if (Build.PRODUCT.startsWith("asop") && Build.VERSION.SDK_INT == 27) {
```
Comment (osma) [inline main/src/cgeo/geocaching/settings/Settings.java:91]: I believe this should be "aosp" not "asop"

Comment (rsudev) [reply main/src/cgeo/geocaching/settings/Settings.java:91]: You are so right...

**Comment by rsudev** [comment]
Retest this please

**Comment by osma** [comment]
Still failing in CI, there is no normal APK to test

**Comment by rsudev** [comment]
I am right now checking it - quite strange.

**Pull request closed without merging**